MyApp/app/helpers/user_helper.rb

```diff
@@ -0,0 +1,9 @@
+module UserHelper
+	def user_status(user)
+		if !user.invitation_accepted_at && user.invitation_sent_at
+			return "Pending"
+		else
+			return "Accepted"
+		end
```

Comment:
correct identation and use shorter syntax
!user.invitation_accepted_at && user.invitation_sent_at ? "Pending" : "Accepted"